defaults/main.yml

```diff
@@ -21,7 +21,7 @@ dell_dsu_gpg_keys:
 # Arguments to DSU
 # Could be changed to for eaxmple only upgrade some categories with "--category=BI" 
```

Comment:
Heyyyy I think this PR changes the "could be changed" to "done!" :) You can remove this comment.